src/infrastructure/database/modules/postgresql/repositories/postgres-assertion.repository.ts

```
@@ -255,4 +256,167 @@
+    if (assertionList.length === 0) {
+      return [];
+    }
+
+    const context = this.createOperationContext('BATCH CREATE Assertions');
+
+    try {
+      // Convert domain entities to database records
+      const records = assertionList.map(assertion => this.mapper.toPersistence(assertion));
+
+      // Use batch insert utility
+      const results = await batchInsert(this.db, assertions as any, records, 'postgresql');
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)